policyengine_us/variables/gov/states/co/tax/income/credits/ctc/co_ctc.py

```diff
@@ -0,0 +1,48 @@
+from policyengine_us.model_api import *
+
+
+class co_ctc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado child tax credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        # C.R.S. 39-22-129. Child tax credit - legislative declaration - definitions.
+        "https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=4f86a852-765a-489a-b554-66f6e7d5b065&nodeid=ABPAACAACAABAACABN&nodepath=%2fROOT%2fABP%2fABPAAC%2fABPAACAAC%2fABPAACAACAAB%2fABPAACAACAABAAC%2fABPAACAACAABAACABN&level=6&haschildren=&populated=false&title=39-22-129.+Child+tax+credit+-+legislative+declaration+-+definitions.&config=014FJAAyNGJkY2Y4Zi1mNjgyLTRkN2YtYmE4OS03NTYzNzYzOTg0OGEKAFBvZENhdGFsb2d592qv2Kywlf8caKqYROP5&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a633G-B1C3-GXJ9-30GW-00008-00&ecomp=7gf59kk&prid=2fe6fa8e-8df8-4d74-a340-c4fa9bc603d5",
```

Comment:
this link isn't working

casetext might be more stable?
https://casetext.com/statute/colorado-revised-statutes/title-39-taxation/specific-taxes/income-tax/article-22-income-tax/part-1-general/section-39-22-129-child-tax-credit-legislative-declaration-definitions-repeal